ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/apps/[externalID]/syncs/Sync/Sync.tsx

```diff
@@ -48,34 +47,25 @@ export function Sync({ externalAppID, syncID }: Props) {
   const { app } = syncRes.data.environment;
   const { sync } = syncRes.data;
 
-  let functions = null;
-  if (sync.status === SyncStatus.Success) {
-    functions = (
-      <FunctionList
-        removedFunctions={sync.removedFunctions}
-        syncedFunctions={sync.syncedFunctions}
-      />
-    );
-    // TODO: Replace with SyncStatus.duplicate after we deploy the API changes
-  } else if (sync.status === 'duplicate') {
-    functions = (
-      <Alert severity="info">
-        This sync is a duplicate because none of the function configurations changed since the
-        previous successful sync. If you would like to view its functions, please navigate to the
-        previous successful sync.
-      </Alert>
-    );
-  }
-
   return (
     <div className="h-full w-full overflow-y-auto">
       <div className="mx-auto w-full max-w-[1200px] p-4">
         {sync.error && <SyncErrorCard className="mb-4" error={sync.error} />}
 
+        {sync.status === 'duplicate' && (
+          <Alert className="mb-4" severity="info">
+            No changes detected since the previous sync. Function bodies may have changed, but those
+            are not inspected when syncing.
```

Comment:
I wonder if there's a better copy for this, since "function bodies" and "inspected" are not terms we are currently using 🤔